src/app/products/_components/ProductCard.tsx

```diff
@@ -0,0 +1,50 @@
+import Icon from '@/components/common/Icon';
+
+interface CardProps {
+  // imgUrl?: string
+  brandName: string; //브랜드명
+  productName: string; //제품명
+  price: string; //가격
+  discountPercentage?: string; //할인률
+}
+
+const ProductCard = ({
+  // imgUrl,
+  brandName,
+  productName,
+  price,
+  discountPercentage,
+}: CardProps) => {
+  const onClick = () => {};
+  return (
+    <div>
```

Comment:
```suggestion
    <>
```
`div`에 특별한 의미나 스타일이 없다면 `<div></div>` 대신 `<></>`을 사용해요 !!
[fragment](https://ko.react.dev/reference/react/Fragment)